liquibase-core/src/main/java/liquibase/changelog/visitor/ValidatingVisitor.java

```diff
@@ -168,7 +169,7 @@ private boolean areChangeSetAttributesValid(ChangeSet changeSet) {
         boolean valid = false;
         if (authorEmpty && idEmpty) {
             validationErrors.addError("ChangeSet Id and Author are empty", changeSet);
-        } else if (authorEmpty) {
+        } else if (authorEmpty && GlobalConfiguration.STRICT.getCurrentValue()) {
```

Comment:
Should the variable name be GlobalConfiguration.AUTHOR_STRICT?

Reply:
Hello @suryaaki2 - it was discussed with Nathan and agreed to use the generic Strict flag.